conformance/system_macros/delta.ion

```diff
@@ -0,0 +1,12 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Test Cases:
+// delta can be invoked by name, address, qualified address, qualified name, binary address, and binary system e-expression op code.
+// the arguments of delta must be non-null integers
```

Comment:
```suggestion
// the arguments of delta must be non-null, unannotated integers
```

Reply:
With other macros (e.g. `make_list`, `make_string`), we silently drop annotations. Do you think we should try to take the same approach for all of the system macros?